src/bundle/DependencyInjection/Configuration/Parser/Notifications.php

```diff
@@ -70,6 +78,13 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
                         ->end()
                     ->end()
                 ->end()
+            ->end()
+            ->arrayNode('notification_count')
```

Comment:
I would have prefer `notifications.count` but it's not compatible with `->useAttributeAsKey('type')` which force count to be seen as a type with only timeout as an available option.